cmd/copyUtil.go

```diff
@@ -171,187 +162,11 @@ func (copyHandlerUtil) generateObjectPath(destinationPath, fileName string) stri
 	return fmt.Sprintf("%s/%s", destinationPath, fileName)
 }
 
-// resourceShouldBeIncluded decides whether the file at given path should be included or not
```

Comment:
Happy to see the fat cut from here. Let's try to find a better place for these other guys soon